<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Система управления культурным центром</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="icon">
        <iconset>
         <normaloff>resourses/desktop.png</normaloff>resourses/desktop.png</iconset>
       </attribute>
       <attribute name="title">
        <string>&amp;Рабочий стол</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="0">
         <widget class="QTableView" name="desktopTableView">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>24</number>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_31">
          <item>
           <widget class="QLabel" name="label_43">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Ваши активные задачи:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_22">
          <item>
           <widget class="QPushButton" name="completeSelectedDesktopPushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Пометить задачи как выполненные (Ctrl+X)</string>
            </property>
            <property name="text">
             <string>Пометить выбранное как &amp;выполненное</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>resourses/check.png</normaloff>resourses/check.png</iconset>
            </property>
            <property name="shortcut">
             <string>Ctrl+X</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="selectedDesktopCountLabel">
            <property name="text">
             <string>0 из 1 выбрано</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1" rowspan="3">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Фильтрация</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_34">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Разновидность:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBox_12">
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
               </property>
               <property name="placeholderText">
                <string>Не выбрано</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="resetDesktopFilterButton">
               <property name="text">
                <string>&amp;Сбросить</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>&amp;Применить</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="requestsTab">
       <attribute name="icon">
        <iconset>
         <normaloff>resourses/work.png</normaloff>resourses/work.png</iconset>
       </attribute>
       <attribute name="title">
        <string>&amp;Заявки</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QTableView" name="worksTableView">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>24</number>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Выберите заявку для редактирования:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="newWorkPushButton">
            <property name="toolTip">
             <string>Создать новую заявку (Ctrl+N)</string>
            </property>
            <property name="text">
             <string>&amp;Новая заявка</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>resourses/create.png</normaloff>resourses/create.png</iconset>
            </property>
            <property name="shortcut">
             <string>Ctrl+N</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="worksTypeMaximizePushButton">
            <property name="toolTip">
             <string>Обзор типов (Ctrl+T)</string>
            </property>
            <property name="text">
             <string>Разновидности</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>resourses/maximize.png</normaloff>resourses/maximize.png</iconset>
            </property>
            <property name="shortcut">
             <string>Ctrl+T</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="editSelectedWorksPushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Отредактировать выбранную заявку (F2)</string>
            </property>
            <property name="text">
             <string>От&amp;редактировать выбранное</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>resourses/edit.png</normaloff>resourses/edit.png</iconset>
            </property>
            <property name="shortcut">
             <string>F2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteSelectedWorksPushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Удалить выбранные заявки (Del)</string>
            </property>
            <property name="text">
             <string>&amp;Удалить выбранное</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>resourses/delete.png</normaloff>resourses/delete.png</iconset>
            </property>
            <property name="shortcut">
             <string>Del</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="selectedWorkCountLabel">
            <property name="text">
             <string>0 из 1 выбрано</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="eventsTab">
       <attribute name="icon">
        <iconset>
         <normaloff>resourses/event.png</normaloff>resourses/event.png</iconset>
       </attribute>
       <attribute name="title">
        <string>&amp;Мероприятия</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QTableView" name="eventsTableView">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>24</number>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_24">
          <item>
           <widget class="QLabel" name="label_32">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Выберите мероприятие для редактирования:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="newEventPushButton">
            <property name="toolTip">
             <string>Создать новое мероприятие (Ctrl+N)</string>
            </property>
            <property name="text">
             <string>&amp;Новое мероприятие</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>resourses/create.png</normaloff>resourses/create.png</iconset>
            </property>
            <property name="shortcut">
             <string>Ctrl+N</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="eventsTypeMaximizePushButton">
            <property name="toolTip">
             <string>Обзор типов (Ctrl+T)</string>
            </property>
            <property name="text">
             <string>Разновидности</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>resourses/maximize.png</normaloff>resourses/maximize.png</iconset>
            </property>
            <property name="shortcut">
             <string>Ctrl+T</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QPushButton" name="editSelectedEventsPushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Отредактировать выбранное мероприятие (F2)</string>
            </property>
            <property name="text">
             <string>От&amp;редактировать выбранное</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>resourses/edit.png</normaloff>resourses/edit.png</iconset>
            </property>
            <property name="shortcut">
             <string>F2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteSelectedEventsPushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Удалить выбранные мероприятия (Del)</string>
            </property>
            <property name="text">
             <string>&amp;Удалить выбранное</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>resourses/delete.png</normaloff>resourses/delete.png</iconset>
            </property>
            <property name="shortcut">
             <string>Del</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="selectedEventsCountLabel">
            <property name="text">
             <string>0 из 1 выбрано</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1" rowspan="3">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Фильтрация</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Пространство:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBox">
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
               </property>
               <property name="placeholderText">
                <string>Не выбрано</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="text">
                <string>Сбросить</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>Применить</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="icon">
        <iconset>
         <normaloff>resourses/location.png</normaloff>resourses/location.png</iconset>
       </attribute>
       <attribute name="title">
        <string>&amp;Локации</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Выберите бронирование для редактирования:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="locationsShowButton">
            <property name="toolTip">
             <string>Обзор типов (Ctrl+T)</string>
            </property>
            <property name="text">
             <string>Места</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>resourses/maximize.png</normaloff>resourses/maximize.png</iconset>
            </property>
            <property name="shortcut">
             <string>Ctrl+T</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="areasOpenButton">
            <property name="text">
             <string>Зоны</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>resourses/categorize.png</normaloff>resourses/categorize.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QPushButton" name="deleteSelectedReservationsPushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Удалить выбранные бронирования (Del)</string>
            </property>
            <property name="text">
             <string>&amp;Удалить выбранное</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>resourses/delete.png</normaloff>resourses/delete.png</iconset>
            </property>
            <property name="shortcut">
             <string>Del</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="selectedReservationsCountLabel">
            <property name="text">
             <string>0 из 1 выбрано</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="reservationsTableView">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>24</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="action_2">
   <property name="icon">
    <iconset>
     <normaloff>resourses/work.png</normaloff>resourses/work.png</iconset>
   </property>
   <property name="text">
    <string>Обзор</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset>
     <normaloff>resourses/categorize.png</normaloff>resourses/categorize.png</iconset>
   </property>
   <property name="text">
    <string>Части</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
